microsoft-edge/devtools-guide-chromium/search/search-tool.md

```diff
@@ -74,10 +74,18 @@ To only find results that match a particular case (lowercase or uppercase charac
 <!-- ====================================================================== -->
 ## Search for regular expressions
 
-You can use regular expressions, including [JavaScript regular expressions](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Regular_Expressions), to find matching results.  To use a regular expression, click the **Use Regular Expression** (`.*`) button in the toolbar and enter a valid regular expression in the search input field.
+You can use regular expressions to find matching results.  To use a regular expression, click the **Use Regular Expression** (`.*`) button in the toolbar and enter a valid [JavaScript regular expressions](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Regular_Expressions) in the search input field.
 
 ![Searching for a regular expression](../media/search-tool/search-tool-regexp.png)
 
+The slash characters that normally delimit regular expressions patterns in JavaScript aren't needed.
+
+When using regular expressions, the **Search** tool matches results by using the `g`, `i`, and `m` regular expression flags:
+
+* The `g` flag makes the search global, meaning that the tool will continue searching files even after a first match has been found.
+* The `i` flag makes the search case-insensitive. To disable this flag and only find results that match a particular case, click the **Match Case** (`Aa`) button in the search toolbar.
+* The `m` flag specifies that the search should be multiline, meaning that the tool treats each line in the source files independently and the `^` and `$` symbols match the start and end of any line, respectively.
+
 
```

Comment:
```suggestion
Examples:

```javascript
regex using g; a string that you'd enter into Search text box
```

```javascript
regex using i; a string that you'd enter into Search text box
```

```javascript
regex using m; a string that you'd enter into Search text box
```



```
Add examples of regex search strings showing g, i, or m. This would be extremely helpful, to concretely see examples, instead of abstractly describing. Any examples would make this much clearer.

Regular expressions > Examples:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#examples

Reply:
I don't think we should add examples here. Users can't use these flags in DevTools. They are set automatically by the tool when you run a regular expression search (by clicking `.*`). The only one you can sort of change is the `i` flag by toggling the `aA` button. But otherwise, `m` and `g` are just always here.
The purpose of this section is to tell users that these flags are enabled for them, by default, so they can understand how their regular expressions get executed.

Reply:
the change to "matches results as if the" addresses this comment